packages/wonder-blocks-dropdown/src/components/single-select.tsx

```diff
@@ -456,10 +455,17 @@ export default class SingleSelect extends React.Component<Props, State> {
             disabled,
         } = this.props;
         const {searchText} = this.state;
-        const allChildren = React.Children.toArray(children).filter(Boolean);
-        // @ts-expect-error [FEI-5019] - TS2345 - Argument of type '(ReactChild | ReactFragment | ReactPortal)[]' is not assignable to parameter of type 'ReactElement<{}, string | JSXElementConstructor<any>>[]'.
+        const allChildren = (
```

Comment:
**praise:** oh great!

Reply:
Got it from the [similar line for MultiSelect](https://github.com/Khan/wonder-blocks/pull/2252/files#diff-f7d0f4de4f6447c1f878dd6c51e6646adae276658cf9d7f29fcea7f630b17430R560)!